[2025-05-24T00:45:32.285+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: punctuality_etl_dag.load_to_postgresql manual__2025-05-24T00:45:29.004889+00:00 [queued]>
[2025-05-24T00:45:32.289+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: punctuality_etl_dag.load_to_postgresql manual__2025-05-24T00:45:29.004889+00:00 [queued]>
[2025-05-24T00:45:32.289+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-05-24T00:45:32.356+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_to_postgresql> on 2025-05-24 00:45:29.004889+00:00
[2025-05-24T00:45:32.359+0000] {standard_task_runner.py:57} INFO - Started process 1156 to run task
[2025-05-24T00:45:32.361+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'punctuality_etl_dag', 'load_to_postgresql', 'manual__2025-05-24T00:45:29.004889+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/punctuality_etl_dag.py', '--cfg-path', '/tmp/tmpxuqtd1gk']
[2025-05-24T00:45:32.363+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask load_to_postgresql
[2025-05-24T00:45:32.390+0000] {task_command.py:415} INFO - Running <TaskInstance: punctuality_etl_dag.load_to_postgresql manual__2025-05-24T00:45:29.004889+00:00 [running]> on host a5551f46ca35
[2025-05-24T00:45:32.433+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='punctuality_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgresql' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T00:45:29.004889+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T00:45:29.004889+00:00'
[2025-05-24T00:45:32.437+0000] {logging_mixin.py:151} INFO - Data loaded successfully!
[2025-05-24T00:45:32.437+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-24T00:45:32.443+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=punctuality_etl_dag, task_id=load_to_postgresql, execution_date=20250524T004529, start_date=20250524T004532, end_date=20250524T004532
[2025-05-24T00:45:32.461+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-24T00:45:32.472+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
